### DISCUSSION TITLE
Visuals w/ incorrect index & fields

### DISCUSSION THREAD

**Discussion by gophsec** [Security-Onion-Solutions/securityonion#10155] [Unsupported Versions]
I noticed quite a few of my dashboard are showing 0 results. When I looked into the visuals I noticed that most are using the wrong index. Most are showing *:Logstash-* or *:log-*. If I correct the index to *:so-* it usually fixes the issue. Sometimes it's filtering on the wrong field. (ie OSSEC - Logs show *:Logstash-* and filtering on 'event_type:ossec' neither which seem to exist.

Are these just legacy visuals and dashboards? Is there a way to update them or do they need to be updated?

**Comment by cm-ops** [comment]
If those dashboards start with `z16.04`, those are legacy dashboards that were included in case someone performed an in-place upgrade from 16.04 to SO2.
